docs/ntppool/use/sample-pool-config.html

```diff
@@ -0,0 +1,8 @@
+<pre class="code">
+driftfile /var/lib/ntp/ntp.drift
+
+pool 2.pool.ntp.org iburst
+pool 3.pool.ntp.org iburst
+pool 0.pool.ntp.org iburst
+pool 1.pool.ntp.org iburst
+</pre>
```

Comment:
There needs to be only one pool directive. With `chronyd` 4 specified pools would result in 16 used servers. With `ntpd`, at least with some versions, I think it would decrease the number of servers as there is a dummy source added for each pool and I suspect it would increase the DNS traffic unnecessarily.

For `ntpd`, there should be `tos maxclock 5` setting to limit the number of used servers to 4.

Reply:
I agree. And `pool pool.ntp.org iburst` would be a great choice, if it weren't for the fact that it has no AAAA-record. Hence, I propose `pool 2.pool.ntp.org iburst`.

Reply:
You are right. Unless `maxsources` is used, `chronyd` will use 4 servers per pool. Smart. :dark_sunglasses: 

Setting `tos maxclock 5` is correct if declaring a single pool. According to the [documentation](https://docs.ntpsec.org/latest/quick.html#pool) of `ntpsec` (a fork of `ntpd`) using only a single `pool` declaration has some limitations. That limitation being that associations are slower to be established, I infer from the comments in the sample configuration.

I haven't used `ntpd` recently, but in `chronyd` having a single pool declared, gets you in sync within a minute.

I'm fine with changing the configuration snippet for `ntpd` to:

```
pool 2.pool.ntp.org iburst
tos maxclock 5
```

I left out the `driftfile` directive. It's distribution specific and I believe taken care of by most distribution's default configuration.

For `chronyd` we need a remark then, that `tos maxclock` is not needed.

Reply:
> There needs to be only one pool directive. With `chronyd` 4 specified pools would result in 16 used servers.

@mlichvar is the number 4 determined by chrony or is it based on the number of addresses returned from a single DNS query of _pool.ntp.org_?  If it's chrony, is that hardcoded or a configurable default value?

Reply:
It's configurable using `maxsources` in connection with `pool`. Default value is 4 and max value is 16.